libs/wingc/src/files.rs

```diff
@@ -75,7 +75,8 @@ impl Files {
 	}
 
 	/// Write all files to the given directory.
-	pub fn emit_files(&self, out_dir: &Utf8Path) -> Result<(), FilesError> {
+	/// If minimize_updates is true, only write files that have changed.
```

Comment:
Performance perhaps?

Reply:
Wait, isn't `minimize_updates` is "less" updates? Maybe the terminology is confusing...

Reply:
It's fewer file modifications, but more syscalls (because we have to both check the stats of each file and possibly write to some of them). See?

I'll just change it :-)